MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -218,6 +219,30 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
     }
 }
 
+// MARK: NavigationMapViewDelegate
+
+extension RouteMapViewController: NavigationMapViewDelegate {
+    
+    func navigationMapView(_ mapView: NavigationMapView, shouldUpdateTo location: CLLocation) -> CLLocation? {
```

Comment:
This will have to do for now, although it’s going to look _really_ weird in Objective-C.

Eventually, as part of mapbox/mapbox-gl-native#6867, we’ll want the MGLMapViewDelegate method upstream to accept an already positioned MGLUserLocation, rather than the raw CLLocationCoordinate2D, so that the heading can also be manipulated.